src/rabbit_policy.erl

```diff
@@ -170,6 +170,11 @@ list(VHost) ->
 list_formatted(VHost) ->
     order_policies(list0(VHost, fun format/1)).
 
+list_formatted(VHost, Ref, AggregatorPid) ->
```

Comment:
why is this code not ordering the policies like `list_formatted/1`? I understand that it has to emit things as they come, but at the same time it has to wait for `rabbit_runtime_parameters:list/2` to produce the whole results, so why not order them as well?